jetstream/src/jsapi_types.ts

```diff
@@ -504,6 +510,24 @@ export const StoreCompression = {
 export type StoreCompression =
   typeof StoreCompression[keyof typeof StoreCompression];
 
+export const PersistMode = {
+  /**
+   * All writes are committed, flushed, and stream data is synced
+   * to disk before the publish acknowledgement is sent. This is the default mode,
+   * and provides the strongest data durability guarantee.
+   */
+  Default: "default",
+  /**
+   * Writes to the stream are committed, but writes to the stream data is synced
+   * asynchronously. Publish acknowledgement is sent before the sync to the store
+   * is completed. There could result in data-loss if the server crashes before the
+   * sync is completed. This provides better performance.
```

Comment:
```suggestion
   * Writes to the stream are committed, but writes to the disk are asynchronously synced.
   * The publish acknowledgement is sent before the sync to the disk is complete.
   * This could result in data-loss if the server crashes before the sync is completed, however
   * with an R3+ stream, the replication provides in-flight redundancy to reduce the likelihood of
   * this occurring with distinct fault domains.
   * This can significantly increase the publish throughput.
```